src/Components/Header/Modal/Modal.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const Modal = ({ width, maxWidth, children, onClose }) => (
+  <ModalOverlay className="modalOverlay" onClick={(e) => onClose(e)}>
```

Comment:
- event 는 자동으로 인자로 들어가기 때문에, 일부러 명시해줄 필요가 없습니다!
```suggestion
  <ModalOverlay className="modalOverlay" onClick={onClose}>
```

Reply:
수정했습니다.
함수가 props로 넘겨준 것을 확인해 주는 방법으로 `() => func()` 형태로 명시해줬는데 불필요한가요?
이렇게 적어주니 props로 넘겨준 함수 호출시 state 변화로 인한 무한 루프를 사전에 방지하기 편하더라고요 ㅎㅎ